modules/common/src/encoding.ts

```diff
@@ -102,6 +102,35 @@ export function decodeRatio(
   return Number(onChainValue) / (10 ** digits);
 }
 
+/** Encodes the particles of a proposalId into a globally unque value for subgraph */
+export function encodeProposalId(pluginAddress: string, id: number) {
+  if (!/^0x[A-Fa-f0-9]{40}$/.test(pluginAddress)) {
```

Comment:
Just a small comment, we have an `isProposalId` function and a `InvalidProposalIdError` on the common package

Reply:
`isAddress` is not in the common package but in the common-client. so it is not usable here. Also the custom error: I followed just how the other functions are working in the common package and all of them use "generic" errors

Reply:
I think isAddress is an ethers.js exported function, you could pull it from there but not super important again

Reply:
okay 👍